.github/workflows/package_dependency_test.yaml

```diff
@@ -0,0 +1,82 @@
+name: package_dependency_test
+on:
+  push:
+    branches: ["master", "*"]
+  pull_request:
+    branches: ["master"]
+jobs:
+  test-centos-stream-9:
+    name: Test on CentOS Stream 9 (Container)
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/centos/centos:stream9
+    steps:
+      - name: Install Deps
+        run: dnf install -y python3-devel python3-setuptools rpm-build python3
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build
+        run: python3 setup.py bdist_rpm --requires=python3-lxml,python3-jinja2 --build-requires=python3-devel,python3-setuptools
+      - name: RPM install
+        run: dnf install -y ./dist/oscap_report-*.noarch.rpm
+      - name: Test parameter -h
+        run: oscap-report -h
+      - name: Test of the basic function
+        run: ./test_of_the_basic_function.sh
+
+  test-fedora-34:
```

Comment:
First of all, F34 would be EOL in couple of months. Not worth setting it up. And, secondly, I don't think we have to test on hard-coded versions of F, `fedora:latest` and `fedora:rawhide` should be enough.